browser/css/calculator.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  font-size: 100%;
```

Comment:
Haha, yeah I had it a little higher then bumped it back down without even thinking that it was useless after that.